app/data/tests/test_views.py

```diff
@@ -156,6 +180,27 @@ def test_arbitrary_filters(self):
         response_data2 = json.loads(self.admin_client.get(url2).content)
         self.assertEqual(len(response_data2), 2)
 
+    def test_created_by(self):
+        url = '/api/records/?details_only=True'
+
+        admin_response_data = json.loads(self.admin_client.get(url).content)
+        record2_result = next(
+            r for r in admin_response_data['results']
+            if r['uuid'] == str(self.record2.uuid)
+        )
+        record3_result = next(
+            r for r in admin_response_data['results']
+            if r['uuid'] == str(self.record3.uuid)
+        )
+
+        self.assertTrue(all('created_by' in result for result in admin_response_data['results']))
+        self.assertEqual(record2_result['created_by'], self.audit_log_entry1.user.email)
+        self.assertEqual(record3_result['created_by'], self.audit_log_entry2.username)
+
+        public_response_data = json.loads(self.public_client.get(url).content)
+        self.assertFalse(all('created_by' in result for result in public_response_data['results']))
```

Comment:
We should split this out into a second test, so they can fail/pass independently of each other.